engine/src/main/java/org/terasology/rendering/dag/stateChanges/DisableDepthMask.java

```diff
@@ -61,10 +61,7 @@ public boolean isTheDefaultInstance() {
 
     @Override
     public boolean equals(Object obj) {
-        if (obj instanceof DisableDepthMask) {
-            return this.enabled == ((DisableDepthMask) obj).isEnabled();
-        }
-        return false;
+        return (obj instanceof DisableDepthMask) && this.enabled == ((DisableDepthMask) obj).isEnabled();
```

Comment:
Eh, agreed on the if maybe being nicer, but it not being a big deal. I guess maybe it is ever so slightly more efficient :-)